examples/p4est_2d_dgsem/elixir_linearizedeuler_gaussian_source.jl

```diff
@@ -0,0 +1,82 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+# Oscillating Gaussian-shaped source terms
+function source_terms_gauss(u, x, t, equations::LinearizedEulerEquations2D)
+  r = 0.1
+  A = 1.0
+  f = 2.0
+
+  # Velocity sources
+  s2 = 0.0
+  s3 = 0.0
+  # Density and pressure source
+  s1 = s4 = exp(-(x[1]^2 + x[2]^2) / (2 * r^2)) * A * sin(2 * pi * f * t)
+
+  return SVector(s1, s2, s3, s4)
+end
+
+initial_condition_zero(x, t, equations::LinearizedEulerEquations2D) = SVector(0.0, 0.0, 0.0, 0.0)
+
+###############################################################################
+# semidiscretization of the acoustic perturbation equations
+
+equations = LinearizedEulerEquations2D(v_mean_global=(-0.5580127018922194, -0.03349364905389032),
```

Comment:
Can you please add a short comment on where these odd values are from here and elsewhere in the elixir?

Reply:
Also, if they are the result of rotating a velocity vector, why not write the rotation here directly instead of the numeric values

Reply:
I clarified this in 752d6810. This is simply an adaptation of the existing `gaussian_source` elixir which uses the acoustic perturbation equations. I decided to rotate the domain (including the mean velocity) so that the flux implementation for curved meshes is properly tested. The values for `c_mean_global` and `rho_mean_global` are slightly off because if they are both `1.0` then some stuff in the fluxes cancels and thus you could argue that in that case they are not tested properly.

Reply:
Thanks, with the changes in your commit it is much clearer. It would be great if you could add a short comment regarding the odd choice of `c_mean_global` and `rho_mean_global` (essentially what you wrote in your comment above) and then resolve this conversation